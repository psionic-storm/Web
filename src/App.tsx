import React, { useEffect } from 'react';
import GlobalStyle from 'Styles/GlobalStyle';
import AppRoutes from 'Routes/AppRoutes';
import { getAllQuotes, getAllReviews } from 'Apis/squareAPI';
import { addBook, getBook, getSpace, updateSpace } from 'Apis/spaceAPI';
import { useDispatch, useSelector } from 'react-redux';
import { requestSignUp } from 'Modules/user/userActions';

function App() {
  localStorage.setItem(
    'accessToken',
    'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6MzUsImxvZ2luSWQiOiJteUlkIiwibmlja25hbWUiOiJteU5pY2tuYW1lIiwiaWF0IjoxNjE1ODY3MjM2fQ.3ldJOQmCB8eQvhmTO-VfRwNRPtz56SBSAGtplWHgJJk',
  );
  const dispatch = useDispatch();
  const body = {
    loginId: 'asddddaddf',
    password: 'gggg',
    nickname: 'asdfdsfsd',
  };

  const user = useSelector((state: any) => state.user);
  useEffect(() => {
    dispatch(requestSignUp(body));
    console.log(user);
  });

  return (
    <>
      <GlobalStyle />
      <AppRoutes />
    </>
  );
}

export default App;
