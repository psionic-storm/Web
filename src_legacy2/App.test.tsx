import React from 'react';
import { render, cleanup } from '@testing-library/react';
import App from './App';

describe('App component', () => {
  afterEach(cleanup);

  it('matches snapshot', () => {
    const { container } = render(<App />);
    expect(container).toMatchSnapshot();
  });

  it('render', () => {
    render(<App />);
  });

  it('umm..', () => {
    // const { getByText } = render(<App />);
    // const asdf = getByText('asdf');
    // expect(asdf).tobeInthe
  });

  it('1+1', () => {
    expect(1 + 1).toEqual(2);
  });
});
