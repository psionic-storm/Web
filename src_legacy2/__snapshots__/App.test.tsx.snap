// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component matches snapshot 1`] = `
<div>
  <p>
    sign up
  </p>
  <input
    placeholder="email.."
    type="text"
    value=""
  />
  <input
    placeholder="password.."
    type="password"
    value=""
  />
  <button>
    signUp btn
  </button>
  <p>
    sign in
  </p>
  <input
    placeholder="email"
    type="text"
    value=""
  />
  <input
    placeholder="password"
    type="text"
    value=""
  />
  <button>
    signIn btn
  </button>
  <br />
  <button>
    getUser btn
  </button>
  <br />
  <button>
    signOut btn
  </button>
  <p>
    default
  </p>
</div>
`;
